cpp/src/arrow/filesystem/s3fs_test.cc

```diff
@@ -1380,5 +1380,33 @@ class TestS3FSGeneric : public S3TestMixin, public GenericFileSystemTest {
 
 GENERIC_FS_TEST_FUNCTIONS(TestS3FSGeneric);
 
+////////////////////////////////////////////////////////////////////////////
+// S3GlobalOptionsDefaults tests
+
+class S3GlobalOptionsDefaults : public ::testing::Test {};
+
+TEST_F(S3GlobalOptionsDefaults, S3GlobalOptionsDefaultsLogLevel) {
```

Comment:
It seems that we don't need to use fixture and simplify test name:

```suggestion
TEST(S3GlobalOptions, DefaultsLogLevel) {
```